packages/website/docs/components/patterns/SelectionAssistant/Selection Assistant.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_class_name: newComponentBadge expComponentBadge
+---
+
+import Input from "../../../components/patterns/SelectionAssistant/InputSelectionAssistant/Basic/Basic.md";
+import TextArea from "../../../components/patterns/SelectionAssistant/TextAreaSelectionAssistant/Basic/Basic.md";
+
+**Note:** The util is in an experimental state
+
+### ES6 Module Import
+
+`import getElementSelection from "@ui5/webcomponents-base/dist/util/SelectionAssistant.js";`
+
+The sample demonstrates the usage of <strong>SelectionAssistant</strong> that enhances user interactions by
```

Comment:
The sample demonstrates the usage of <strong>SelectionAssistant</strong>, which enhances user interactions by